api/src/http.rs

```diff
@@ -420,9 +420,9 @@ impl Default for ProxyConfig {
 /// Configuration for mirror.
 #[derive(Clone, Deserialize, Serialize, Debug)]
 pub struct MirrorConfig {
-    /// Mirror url host.
```

Comment:
Please add some docs in [here](https://github.com/dragonflyoss/image-service/blob/master/docs/nydusd.md#common-fields-in-config)

Reply:
Finished.